pytket_qir/gatesets/base.py

```diff
@@ -1,17 +1,19 @@
+from dataclasses import dataclass
 from enum import Enum
 from string import Template
 from typing import Callable, Dict, List, NamedTuple, Union
 
-from pyqir.generator.types import Double, Integer, Qubit, Result, Void  # type: ignore
+from pyqir.generator.types import Double, Int, Qubit, Result, Void  # type: ignore
 
 
-PyQirInputTypes = Union[Double, Qubit, Result]
-PyQirOutputTypes = Union[Integer, Result, Void]
+PyQirParameterTypes = Union[Double, Qubit, Result]
+PyQirReturnTypes = Union[Int, Result, Void]
```

Comment:
The names would be better as singular:
```suggestion
PyQirParameterType = Union[Double, Qubit, Result]
PyQirReturnType = Union[Int, Result, Void]
```

Reply:
Thanks.